Virtusize/Sources/VirtusizeRepository.swift

```diff
@@ -195,11 +195,11 @@ internal class VirtusizeRepository: NSObject {
  		}
 
 		if let userBodyProfile = userBodyProfile {
-			bodyProfileRecommendedSizes = VirtusizeAPIService.getBodyProfileRecommendedSizesAsync(
+            bodyProfileRecommendedSize = VirtusizeAPIService.getBodyProfileRecommendedSizesAsync(
 				productTypes: productTypes!,
 				storeProduct: storeProduct,
 				userBodyProfile: userBodyProfile
-			).success
+            ).success?.first
```

Comment:
The only change needed to be made compared to https://github.com/virtusize/integration_ios/commit/009be7c8f6d7a115fd5893f7bac889c79b0b0667#diff-b0d39844a5b2e0fb7805d14e6424a7a52be5e82def71b561db7c65f1747e476c was that we could just assigned the first item of BodyProfileRecommendedSizeArray to bodyProfileRecommendedSize.

Reply:
@akueisara Yes exactly. The new Size recommendation API (called SRAPI) actually return an array of only 1 object. See here